sites/NCG-INGRID-PT.yaml

```diff
@@ -19,3 +19,15 @@ vos:
 - auth:
     project_id: a53ca78c534046e5b13f4537ae698411
   name: worsica.vo.incd.pt
+- auth:
+    project_id: 05e52356addc44e18ef2bd14f2e2f67d
+  name: covid-synergy
+- auth:
+    project_id: a53ca78c534046e5b13f4537ae698411
+  name: worsica
```

Comment:
This was already above

Reply:
Ok; please reject this PR, I'll create another one later today.

Reply:
Actually, this PR should work now.